docs/ENVIRONMENT.md

```diff
@@ -56,6 +56,11 @@ All tools:
   This is useful for debugging and testing initrd-only programs in the main
   system.
 
+* `$SYSTEMD_INITRD_IS_RAMFS=1` — take a boolean. If set to false, systemd
```

Comment:
i'd prefer if we'd extend $SYSTEMD_IN_INITRD instead. right now it takes a boolean only. i think it would make sense to extend it to take a new special value "lenient" or "lax" or so, which means: do the usual check, but weaker, i.e. without the ram fs check.

maybe even add "auto" which implements the current default behaviour, in case anyone wants to request that explicitly... that way:

1. SYSTEMD_INITRD_IS_RAMFS=0/false/off/no → definitely not in initrd
2. SYSTEMD_INITRD_IS_RAMFS=1/true/on/yes → definitely in initrd
3. SYSTEMD_INITRD_IS_RAMFS=auto → the current logic, and default logic: check for /etc/initrd-release + temporary fs
4. SYSTEMD_INITRD_IS_RAMFS=lenient → the new logic, where only /etc/initrd-release is checked

I think this would be a nice, natural extension, and be prettier than adding yet another top-level env var for this

Reply:
Good suggestion, I'll update it.